### DISCUSSION TITLE
Requested Screenshots

### DISCUSSION THREAD

**Discussion by Budlyte** [advplyr/audiobookshelf#60] [Show and tell]
I'll wait on screenshots of the Series Tab for now

Library View - Sorted by Most Recent Date Added
![image](https://user-images.githubusercontent.com/4451365/134827097-b6295326-a676-4302-b13e-9b2da6a7e4f2.png)

Viewing Author - Sorted by Volume # Ascending - Showing list of sorting options
![image](https://user-images.githubusercontent.com/4451365/134827242-33a915e8-7116-40a4-be15-446533fc1c4f.png)

Using Keyword Filter - Sorting by Author (First Last) Descending - Showing available Filter Options
![image](https://user-images.githubusercontent.com/4451365/134827300-73e4f82b-9c5e-43f9-aee5-994dbd64a1ab.png)

Series Filter Options
![image](https://user-images.githubusercontent.com/4451365/134827324-27f786ce-c1ec-4252-b44b-06ad59921ee2.png)

Authors Filter Options
![image](https://user-images.githubusercontent.com/4451365/134827342-082f351d-d5c7-41cc-b323-12522314c6f3.png)

Progress Filter Options
![image](https://user-images.githubusercontent.com/4451365/134827351-9008e2bf-9c61-46fa-87c1-304aac0809a2.png)

Showing Marked Read items - Sorted by Author (Last, First) Ascending
![image](https://user-images.githubusercontent.com/4451365/134827401-eea53fff-4cf6-4be5-90a4-49763d879aaf.png)

Showing Unread Items - Sorted by Data Added Ascending (Most Recent)
![image](https://user-images.githubusercontent.com/4451365/134827442-03245d8f-aff5-4e80-9950-09f79a0af9c5.png)

Showing In Progress
![image](https://user-images.githubusercontent.com/4451365/134827536-27ebe592-bc3b-4bdc-a7e2-6fc46f546c22.png)

Using the Searchbar
![image](https://user-images.githubusercontent.com/4451365/134827575-eea146f5-e6e7-48bf-a819-371e466a7652.png)

**Comment by advplyr** [comment]
Beautiful, thank you

**Comment by advplyr** [comment]
https://www.audiobookshelf.org/showcase

**Comment by Budlyte** [comment]
Shots of that Series View.

Viewing first opening of shelf.
![image](https://user-images.githubusercontent.com/4451365/135764312-9dfd9aa7-951b-433c-979f-6559b3343286.png)

View of item being moused over, and a closer look at the In-Progress progress bar.
![image](https://user-images.githubusercontent.com/4451365/135764330-3578c70b-1dc9-42bd-aea0-61a028f057c2.png)

View inside of Series
![image](https://user-images.githubusercontent.com/4451365/135764380-3dc0ee32-9c2b-4e2e-a2b5-f1691b49d132.png)

Using the Keyword Filter 
![image](https://user-images.githubusercontent.com/4451365/135764406-30c55a9e-fdc8-41e4-94c8-54b621097d52.png)

**Reply by advplyr** [reply]
Thanks, looking really clean, just updated the showcase.

**Comment by Budlyte** [comment]
How you doing on screenshots? There's been a lot of discussion over new features, with people posting screens of what they got going on, but figured I'd stop by and see if you need anything?  Seems like attention on your project has really taken off, as it should since you're awesome and what you've built is amazing.

**Reply by advplyr** [reply]
Thanks! I took out the old out-dated screenshots from the showcase page https://www.audiobookshelf.org/showcase
but we could probably improve that still, a lot of features have been added since.

**Comment by hybridtracer** [comment]
If you need any screenshots of libraries not using square covers let me know as I've got my library setup with "Book" size covers.

**Reply by advplyr** [reply]
Yeah that could be a good addition to the showcase. If you want to share some that would be great.

**Reply by hybridtracer** [reply]
Here are a couple of Library and Series view. If there are any specific ones let me know.

![Capture](https://user-images.githubusercontent.com/20684104/180664469-3dd5f136-07c3-449b-aa9c-1a4189822866.JPG)
![Series](https://user-images.githubusercontent.com/20684104/180664470-ce46e3c6-4a89-49ba-9519-0750fa46eac4.JPG)
